.vscode/settings.json

```diff
@@ -6,6 +6,9 @@
       "go.toolsEnvVars": {
             "GOFUMPT_SPLIT_LONG_LINES": "on"
       },
+      "go.delveConfig": {
```

Comment:
for now this fixes my issue with delve, let's check later if there are other solutions